content/en/gitops.md

```diff
@@ -1,29 +1,38 @@
 ---
 title: GitOps
-status: Feedback Appreciated
+status: Completed
 category: Concept
 tags: ["methodology", "", ""]
 ---
 
-GitOps is a set of best practices based on [shared principles](https://opengitops.dev/), 
-applied to a workflow that depends on software agents that 
-enable automation to reconcile a declared system state or configuration in a git repository.
-These software agents and practices are used to execute a cohesive workflow that 
+GitOps is a set of best practices for managing software systems  
+from the beginning of the development workflow, all the way to deployment
+based on [shared (OpenGitOps) principles](https://opengitops.dev/).
+
+GitOps requires the desired state or configuration of the system to 
+be stored in a source control system like Git.
+These practices are then used to execute a cohesive workflow that 
 leverages a source control system like Git as the “single source of truth” and 
 extends this practice to applications, infrastructure, and operational procedures.
 
 ## Problem it addresses
 
-Existing processes for infrastructure configuration management can face challenges 
-such as configuration drift, failed deployments, relying on a system's previous state for success, 
-missing documentation, or unknown development history.
-Adopting a GitOps workflow can help alleviate these issues, among several others.
+Existing processes for configuration management can face 
+challenges such as configuration drift, failed deployments, inconsistency 
+across different infrastructure, or unknown development history.
+Emphasising on continues deployment and delivery, 
+a GitOps workflow can help alleviate these issues, 
+among several others.
```

Comment:
```suggestion
Existing processes for configuration management can face challenges such as
configuration drift, failed deployments, inconsistency across different infrastructure,
and unknown development history. By emphasizing continuous deployment and
delivery, GitOps helps alleviate these issues.
```

Reply:
Approved but I cannot signoff and commit through the webui, it says it cannot apply suggestions on deleted lines ^^